layouts/partials/site-header.html

```diff
@@ -27,7 +27,17 @@ <h1 class="font-extrabold text-3xl md:text-5xl *:block">{{ . | markdownify }}</h
                   {{ end }}
               </div>
               <div class="mx-auto">
-                <img src="img/vertical-logo.png" alt="Rotational Labs logo" class="mt-4 sm:mt-[4.5rem] h-16 sm:h-40 md:h-64 w-auto">
+                <picture>
+                    {{ $logo := resources.Get "img/vertical-logo.png" }}
+                    {{ $logo := $logo.Process "webp q80" }}
+
+                    {{ $tiny := $logo.Resize "x64" }}
+                    {{ $small := $logo.Resize "x160" }}
+                    {{ $medium := $logo.Resize "x256" }}
+                    <source srcset="{{ $medium.RelPermalink }}" media="(min-width: 768px)">
```

Comment:
Instead of setting height on the image with CSS, the image has been resized with the height values that were previously specified. We'll now use the `picture` element to help determine which version of the image should be loaded depending on the width of the user's device.

Reply:
It does look like some of the blog post images are stretched tall because of the fixed height; is there a way to use cropping instead of stretching on the images? Or can we write a script to determine which images do not have the correct aspect ratio? I have written a python script that can crop images into the correct aspect ratio that we can use to fix the stretching.

Reply:
Hugo included a crop option as well. I'll take a look to see if that'll help. I think this mostly occurred with images in portrait mode instead of landscape.